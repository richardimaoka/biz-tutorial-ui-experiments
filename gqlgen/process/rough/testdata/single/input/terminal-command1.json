{
  "step": "66105871-d23c-4e54-98f4-b715b330a0da",
  "phase": "CMD の動作理解",
  "comment": "",
  "type": "terminal command",
  "instruction": "mkdir myproj",
  "tooltip": "チュートリアル用ディレクトリを作成します"
}
