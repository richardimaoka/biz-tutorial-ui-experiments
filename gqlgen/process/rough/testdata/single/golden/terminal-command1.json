[
  {
    "fromRoughStep": true,
    "subId": "terminalCommandStep",
    "parentStep": "66105871-d23c-4e54-98f4-b715b330a0da",
    "step": "",
    "focusColumn": "Terminal",
    "column1": "Terminal",
    "terminalType": "command",
    "terminalText": "mkdir myproj"
  }
]